protos/BUILD

```diff
@@ -60,3 +60,15 @@ ts_proto_library(
         ":server_proto",
     ],
 )
+
+load("//testing:build_test.bzl", "build_test")
+
+build_test(
+    name = "build_test",
+    deps = [
```

Comment:
should probably really have the proto_library rules here, the other ones are useful but not as necessary.